nixos/tests/podman/default.nix

```diff
@@ -1,5 +1,22 @@
 import ../make-test-python.nix (
-  { pkgs, lib, ... }: {
+  { pkgs, lib, ... }:
```

Comment:
I'm not too deep into podman internals, but fair enough. Do we maybe have another, "more common" user generators that would now work (and previosuly didn't)?

Some quick googling only mentioned `xdg-autostart-generator`, and I have no clue if this currently works (or not), and how…

Reply:
`nix-locate` gives us this output:

```
~ ➜  nix-locate --top-level user-generators/
systemd.out                                      36,744 x /nix/store/9cxd17xnmw0bi8n4nf722ysqj2bjlh8s-systemd-255.4/lib/systemd/user-generators/systemd-xdg-autostart-generator
systemdMinimal.out                               36,744 x /nix/store/yhnhwqm09w9302rx24804i67jfp8ar4h-systemd-minimal-255.4/lib/systemd/user-generators/systemd-xdg-autostart-generator
systemdUkify.out                                 36,744 x /nix/store/swdvrw38ah73lbvklr972kfjw540r3df-systemd-255.4/lib/systemd/user-generators/systemd-xdg-autostart-generator
podman.out                                            0 s /nix/store/cppmivpa5ipbl56nlvnm1cwm3iq0vpzz-podman-5.0.3/lib/systemd/user-generators/podman-user-generator
```

I'm not sure about `systemd-xdg-autostart-generator` too.

Reply:
Strangely, this binary does not get linked into `/etc/systemd/user-generators` :thinking:

Reply:
In any case, this is a strict improvement, so let's get it in!